src/transformers/utils/import_utils.py

```diff
@@ -577,7 +577,10 @@ def is_optimum_available():
 
 
 def is_safetensors_available():
-    return importlib.util.find_spec("safetensors") is not None
+    if version.parse(_torch_version) >= version.parse("1.10"):
```

Comment:
should we check if torch is available before doing `version.parse(_torch_version)`. Otherwise, I am not sure what will happen (for `version.parse`) with `_torch_version = "N/A"`.

Reply:
ah, the usual gotcha for me! yes! thank you, @ydshieh

Reply:
cc @Narsil 🙏

Reply:
which one?

A. 
```
def is_safetensors_available():
    if is_torch_available():
        if version.parse(_torch_version) >= version.parse("1.10"):
            return importlib.util.find_spec("safetensors") is not None
        else:
            return False
    else:
        return importlib.util.find_spec("safetensors") is not None
```

B. 
```
def is_safetensors_available():
    if is_torch_available() and version.parse(_torch_version) >= version.parse("1.10"):
        return importlib.util.find_spec("safetensors") is not None
    else:
        return False
```

Reply:
But personally, I think a (ugly) hack is: if 

torch is not available ==> return importlib.util.find_spec("safetensors") is not None
torch is avaialalbe + version.parse(_torch_version) >= version.parse("1.10") ==> same

otherwise:

return False

Reply:
so A. this is what I have just pushed. please check if it works for you.

Reply:
works for me, and since Sylvain approved, you can go ahead once CI is green.

(I personally don't know if `safetensors` is relevant to `tensorflow` - I heard about it once, but it might be relevant just in future)

Reply:
It is relevant to TensorFlow as well as PyTorch. safetensors weights are fraework-agnostic.